### ISSUE TITLE
Unable to display the meetings. Please check the url of the bigbluebutton server AND check - URL is correct - BUG ?

### ISSUE THREAD

**Issue by lrit18** [albertosgz/Wordpress_BigBlueButton_plugin#39]
Hello everybody,
I always get the message "Unable to display the meetings. Please check the url of the bigbluebutton server AND check to see if the bigbluebutton server is running."
My BBB URL is correct and the meetings are working too. My url is https://xx.yy.net/bigbluebutton/
Can someone help me how this message disappears? Thanks.

**Comment by albertosgz** [comment]
I think that message is a known issue. It means there is no meetings running in the server (at least configured by the plugin).
Can that make sense?

**Comment by lrit18** [comment]
Yes, the message disappears when a meeting (created by the plugin) is running. "List of Active Meeting Rooms in BBB Server" then appears at the bottom. However, the meetings are not displayed in the list although it is active.

**Comment by albertosgz** [comment]
Only those created by the plugin will be listed, if I remember well.

**Comment by lrit18** [comment]
This meetings was created by the plugin.

**Comment by albertosgz** [comment]
And the plugin was not removed and installed in the meantime? Did you see any error in the php logs?
Sounds like a bug.

**Comment by lrit18** [comment]
I installed the plugin and then manually created the rooms. Then I started it with the "Join" button. See picture atached.

![bbb](https://user-images.githubusercontent.com/20736613/115865413-90d60100-a438-11eb-9213-e885a9c06cf6.JPG)

It also looks a bit strange because the column headings are shown twice

**Comment by albertosgz** [comment]
it should work as usual. Do you have some error on php logs or dev console?

Alberto Sánchez González

**Comment by lrit18** [comment]
![bbb1](https://user-images.githubusercontent.com/20736613/115887884-a821e800-a452-11eb-9194-0c5ea7fd4bfc.JPG)
I found this in firefox console.

**Comment by lrit18** [comment]
![bbb4](https://user-images.githubusercontent.com/20736613/115888299-0e0e6f80-a453-11eb-8161-2123cfffe46c.JPG)

**Comment by lrit18** [comment]
Did you do magic Suddenly, without doing anything, the plugin page now looks better :) the pdf, csv, e.g. buttons for export are there

**Comment by albertosgz** [comment]
The looking is rendered by the datatables library, so if can't load well, everything is broken

**Comment by lrit18** [comment]
What do you mean by "everything is broken". The plugin? The file https: //www.xxxxx.xx/wp-content/plugins/bbb-administration-panel//DataTables/datatables.min.js?ver=5.6 can be called up without any problems if I enter it in the browser's address line.

**Comment by lrit18** [comment]
I have now installed a fresh WordPress. Then I installed the BBB Administration Plugin for the first time. Then I created 2 rooms in the plugin. I think the problems have to do with the current version of jQuery. I get a message that something is wrong with jQuery in the BBB Administration Plugin. See screenshots again. thanks
![bbb error](https://user-images.githubusercontent.com/20736613/115951291-490ab480-a4e0-11eb-9038-823553ed4f87.JPG)
![bbb error 1](https://user-images.githubusercontent.com/20736613/115951296-4c05a500-a4e0-11eb-837d-ccd1fab604d2.JPG)

**Comment by albertosgz** [comment]
Hi @lrit18 , I just test it myself with a fresh installation of WP 5.8.3, and I don't such error on console.
Is still happening?
